src/functions/template-tags/venue.php

```diff
@@ -725,14 +725,16 @@ function tribe_get_phone( $postId = null ) {
 	 * @since ??
 	 * @since 4.5.11 Added docblock and venue ID to filter
 	 *
-	 * @param bool $output The escaped phone number for the venue.
-	 * @param int  $venue_id The venue ID
+	 * @param string $output The escaped phone number for the venue.
+	 * @param int    $venue_id The venue ID
 	 */
 	return apply_filters( 'tribe_get_phone', $output, $venue_id );
 }
 
 /**
  * Get all the venues
+ * 
+ * @since TBD - Applied the `tec_events_custom_tables_v1_normalize_occurrence_id` filter to convert provisional IDs into regular IDs.
```

Comment:
Treat this like a param, etc for formatting (no dash)
```suggestion
 * @since TBD Applied the `tec_events_custom_tables_v1_normalize_occurrence_id` filter to convert provisional IDs into regular IDs.
```